### ISSUE TITLE
Linear model weight parameters (CUDA) are not included in state_dict()

### ISSUE THREAD

**Issue by SHAWNandSHIRLEY** [shenweichen/DeepCTR-Torch#87]
**Describe the bug**
Linear model weight parameters are not included in state_dict() after sending to CUDA. I found this problem when loading the serialized model parameters with `torch.save(model.state_dict(), PATH)` 

**To Reproduce**
The tensor is sent to other device after creating `nn.Parameter`, and so the returned instance is not a trainable tensor anymore. To avoid this, the `to_(device)` should be called first when creating a tensor, followed by creating `nn.Parameter`.

https://github.com/shenweichen/DeepCTR-Torch/blob/bb6064343e8d271843bbd7efb9e8f3ecbd5d42b3/deepctr_torch/models/basemodel.py#L52-L53

Similar issue can be found in pytorch forum.
 [https://discuss.pytorch.org/t/cuda-parameter-not-included-in-state-dict/30801](url)

**Operating environment:**
 - python version 3.7
 - torch version 1.4.0
 - deepctr-torch version 1.0.3

**Comment by ZhangYuef** [comment]
Thanks for pointing out! This should be fixed as 
```python
self.weight = nn.Parameter(torch.Tensor(sum(fc.dimension for fc in self.dense_feature_columns), 1).to(device) )
```
Wait for future pull request to fix it.

**Comment by zanshuxun** [comment]
fixed

**Comment by zanshuxun** [comment]
Hi, we have solved this issue in [v0.2.4](https://github.com/shenweichen/DeepCTR-Torch/releases/tag/v0.2.4), please use `pip install -U deepctr-torch` to upgrade.
